libs/datasets/timeseries.py

```diff
@@ -396,7 +396,10 @@ def _merge_attributes(df1: pd.DataFrame, df2: pd.DataFrame) -> pd.DataFrame:
         _log.warning(f"Re-adding empty columns: {missing_columns}")
         wide = wide.reindex(columns=[*wide.columns, *missing_columns])
     # Make all non-GEO_DATA_COLUMNS numeric so that aggregation functions work on them.
-    numeric_columns = list(all_columns - set(GEO_DATA_COLUMNS))
+    df1_numeric_columns = df1.columns[df1.dtypes.apply(is_numeric_dtype)]
```

Comment:
I'm not yet a fan of this change. L 403 `apply(pd.to_numeric` is there so we notice when something non-numeric slips in to a column that is expected to be floats. This change will let such values go deeper in where they break other code in hard to debug ways. I'd like to stick with explicitly listing columns that can be non-numeric for now.

Reply:
👍  is it too hacky to just include this column with a TODO to generalize better to non-numeric columns

Reply:
hard-coding removing the new column from numeric_columns is totally fine with me. It seems less likely to cause a problem than depending on the dtypes.

Reply:
Fixes some tests too 😅